data/shaders/core.frag

```diff
@@ -1,23 +1,37 @@
 #version 330 core
 
-out vec4 fragColor;
-
 //DEFINES
 
-uniform mat4 colorMatrix;
-
 in vData {
 	vec3 lightDir;
 	vec2 texCoord;
 	vec3 normal;
 	vec4 color;
 } fragIn;
 
+out vec4 fragColor;
+
+layout (std140) uniform shaderMatrices {
+	mat4 projMatrix;
+	mat4 mvMatrix;
+	mat4 normalMatrix;
+	mat4 colorMatrix;
+} shaderMatrices;
```

Comment:
![schermafdruk van 2018-11-01 19-31-46](https://user-images.githubusercontent.com/2794331/47871757-2a466e00-de0d-11e8-8476-b153f2fc75bd.png)
this is "already defined"

Reply:
What the blue fuck? Maybe I need to change the instance names between stages? 🤷🏼‍♂️